.github/workflows/ci.yml

```diff
@@ -6,34 +6,37 @@ on:
       - master
   pull_request:
     branches:
-      - '**'
```

Comment:
my editor / vscode reformatted this file if this is not acceptable, I can revert! I have to check what settings exist that are doing this

Reply:
This shouldn't be an issue. If this is the only change, then you can merge with this change since both versions are syntactically correct.

Reply:
FWIW, I don't think the "**" is needed here. Just having `pull_request` under `on` should suffice to get the workflow to run on all Paragon PR updates.